### PR TITLE
3D Inversive Geometry Experiments

### PR DESCRIPTION
**Pull request by ptrgags**
Currently I'm experimenting with some of the new transformations. 

Things remaining to do:
- [ ] Create parameter files for the rest of the transformations I have listed in my notebook
    - [x] 3D spirals
    - [ ] Hyperbolic: Needs better display settings
    - [ ] Parabolic
    - [ ] Rotate through ring
    - [x] Double Rotation
    - [ ] Loxodromic: Needs better display settings 
    - [x] Axis-aligned Loxodromic
    - [ ] Conjugated Screw
    - [ ] Axis-aligned Screw
- [ ] Debug the `+inverse` shortcut. Something's not working properly.
- [x] Try to add a unit sphere to the viewer
- [ ] Display transformation parameters when `std::env::var("DEBUG_XFORMS")` = true
- [ ] Make a better wrapper script that only needs to specify the name of a parameter file

### CODE REVIEW

**Comment by rkr35** [comment]
Test

File: src/algorithms.rs [hunk 1]
```diff
@@ -88,6 +90,14 @@ impl Algorithm for ChaosGame {
 
             pos = self.position_ifs.transform(&pos);
             color_vec = self.color_ifs.transform(&color_vec);
+
+            // Show progress every 100K iterations
```
Comment (ptrgags) [inline src/algorithms.rs]:
```suggestion
            // Show progress every UPDATE_FREQ iterations
```

**Review by ptrgags — COMMENTED** [review]
Looks good overall, just one small tweak.

There's plenty more that could be better about this program (including adding a README!) but not for this PR. I want to revisit this program once CesiumJS has an implementation of `EXT_feature_metadata`

**Comment by ptrgags** [comment]
Updated this PR, but when I tried to run things, I was getting errors about the transforms needing to be a string... Maybe I changed some behavior without changing the parameter files? I'll investigate this another time.

**Comment by ptrgags** [comment]
Merging this so I can work on some new features as described in `logbook.md`

**Pull request merged**